### PR TITLE
display linked projects and linked releases hierarchically in project…

### LINKED ISSUE #158: Rework hierarchy of linked projects (project detail view)
At linked projects:
* tree view
* linked sub projects
* sub projects of sub projects, etc ...
* directly referenced releases at every project (including sub projects)
* release info: vendor, component, release version
* link to go to release
* no linked releases of releases

At linked releases
* no high priority
* at second stage
* linked releases of directly referenced releases for a project
* not considering releases of sub projects (or sub sub or sub sub sub projects ...)
* could be same as with linked releases in release detail view, however not just for the release of release detail view, but for all directly referenced releases of this project.

adding: the releases in the listing(s) should have also the component type there: When components (releases!) are shown in projects, there should be the components type as well in the listing. icon would be better than text.

### PR DESCRIPTION
**Pull request by alex-evo**
… detail view and release detail view

closes  #158

### CODE REVIEW

**Comment by mcjaeger** [comment]
please
1) change the license header as done in the past two commits
2) add vendor (see ticket) to release info line

**Comment by mcjaeger** [comment]
tested-by:michael.c.jaeger@siemens.com

File: frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+table.treetable span.indenter {
```
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css]: please add header for small files, such as in:
https://github.com/sw360/sw360portal/blob/master/frontend/sw360-portlet/src/main/webapp/css/search.css

File: backend/src/src-components/src/test/java/com/siemens/sw360/components/db/ProjectDatabaseHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
+/*
```
Comment (mcjaeger) [inline backend/src/src-components/src/test/java/com/siemens/sw360/components/db/ProjectDatabaseHandlerTest.java]: epl-1.0 please

File: frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css (hunk 1 above)
Comment (alex-evo) [inline frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css]: These two new css files come from jquery treetable plugin. Do you think I should copy the header from jquery.treetable.js?

**Comment by alex-evo** [comment]
>2) add vendor (see ticket) to release info line

In Linked Projects, vendor (if assigned) is displayed together with the release name. Do you want me to separate it into its own column?

**Comment by mcjaeger** [comment]
I think I was wrong about the vendor and I took a component without a vendor. I was just messing up with the test data I have (quickly) entered. Sorry for my mistake.

File: frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css (hunk 1 above)
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css]: then please add: https://github.com/sw360/sw360portal/blob/master/frontend/sw360-portlet/src/main/webapp/css/dataTable_SiemensHomepage.css

File: frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.theme.default.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/*table.treetable {*/
```
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.theme.default.css]: same as  frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css

File: frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css (hunk 1 above)
Comment (alex-evo) [inline frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css]: Did I get it right this time?

**Pull request merged**